<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditClientWidget</class>
 <widget class="QWidget" name="AuditClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AuditClientWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Connexion au serveur</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Adresse IP du serveur :</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditAdresseIP"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Num√©ro de port du serveur :</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditPort"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButtonQuitter">
              <property name="text">
               <string>Quitter</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonConnexion">
              <property name="text">
               <string>Connexion</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Etat de la connexion</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEditEtat"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxInformationsClients">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Informations Clients</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButtonOrdinateur">
            <property name="text">
             <string>Nom de l'ordinateur distant</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonUtilisateur">
            <property name="text">
             <string>Nom de l'utilisateur</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonArchitecture">
            <property name="text">
             <string>Architecture processeur</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonOs">
            <property name="text">
             <string>OS de l'ordinateur</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLineEdit" name="lineEditOrdinateur"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditUtilisateur"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditArchitecture"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditOs"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>AuditClientWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
